linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml

```diff
@@ -11,6 +11,10 @@ description: |-
 {{% if product in ["rhel6", "rhel7"] %}}
     Red Hat Enterprise Linux is supported by Red Hat, Inc. As the Red Hat Enterprise
     Linux vendor, Red Hat, Inc. is responsible for maintaining government certifications and standards.
+{{% elif product in ["sle12"] %}}
+    SUSE Enterprise Linux is supported by SUSE Software Solutions Germanu GmbH. As the SUSE Enterprise
+    Linux vendor, SUSE Software Solutions Germanu GmbH is responsible for maintaining government
+    certifications and standards.
```

Comment:
```suggestion
{{% elif product in ["sle12", "sle15"] %}}
    SUSE Enterprise Linux is supported by SUSE Software Solutions Germanu GmbH. As the SUSE Enterprise
    Linux vendor, SUSE Software Solutions Germanu GmbH is responsible for maintaining government
    certifications and standards.
```

Reply:
Should there be a similar text for ``sle15``?

Reply:
My reasoning  behind leaving it off was that SLE15 has not yet achieved FIPS certification (according to their website).  I reasoned that is why there was no entry for RHEL8 in that same block.  If that is mistaken, I will gladly add it in.

Reply:
@eradot4027 Makes sense! Your reasoning is spot on. I don't track SLE15 so unaware of what FIPS has been achieved, so appreciate your keeping everyone accurate on this.